packages/zoe/src/zoe.js

```diff
@@ -543,7 +543,12 @@ const makeZoe = (additionalEndowments = {}) => {
               // Once the contract is made, we add the publicAPI to the
               // contractRecord
               instanceTable.update(instanceHandle, { publicAPI });
-              return invite;
+              return inviteIssuer.isLive(invite).then(success => {
+                if (success) {
```

Comment:
Can we use an assert here and reverse the return and the error check?
```suggestion
assert(success, details`invites must be issued by the inviteIssuer.`);
return invite;
```

Reply:
done.